pages/common/unzip.md

```diff
@@ -1,24 +1,25 @@
 # unzip
 
-> Extract compressed files in a ZIP archive.
+> Extract compressed files/directories from .zip archives.
```

Comment:
Files in a ZIP archive aren't necessarily compressed (and directories can't be).

```suggestion
> Extract files/directories from ZIP archives.
```